containers/podman-desktop-extension-ai-lab
# use ai-lab-recipes v1.1.0 in catalog

### What does this PR do?

use ai-lab-recipes v1.1.0 in catalog


### What issues does this PR fix or reference?


### How to test this PR?

Start a recipe, go to the sources and check that the current commit is at tag v1.1.0
